Portal/Controllers/WasteController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Portal.Models;
+using Portal.Services.Interfaces;
+
+namespace Portal.Controllers
+{
+    public class WasteController : Controller
+    {
+
+        private readonly IWasteService _wasteService;
+
+        public WasteController(IWasteService wasteService)
+        {
+            _wasteService = wasteService;
+        }
+
+        [HttpGet]
+        public IActionResult DuringWhichMonth(int? id)
+        {
+            if (id == null)
+                throw new ArgumentNullException("id");
+
+            var model = _wasteService.GetCurrentQuarter(id.Value);
+
+            return View(model);
+        }
+
+        [HttpPost]
+        public IActionResult DuringWhichMonth(DuringWhichMonthRequestViewModel monthSelected)
```

Comment:
I reckon i friendlier, punchier name could be used for the route. Perhaps just "Month" or something. You can keep the function name, and just add an attribute like the following:
`[Route("Month")]`